.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -434,8 +438,10 @@ tasks:
 <% for(const group of E2E_TEST_GROUPS) { %>
   - name: test-packaged-app-<%= group.number %>
     tags:
-    - required-for-publish
-    - run-on-pr
+      - required-for-publish
+      - run-on-pr
+      - assigned_to_jira_team_compass_compass
+      - foliage_check_task_only
```

Comment:
any tests that fail in this task, a single issue will be opened (as opposed to opening ticket per failing test)